prac_07/programming_language.py

```diff
@@ -0,0 +1,51 @@
+"""
+CP1404/CP5632 Practical - Suggested Solution
+Programming Language class with tests.
+"""
+
+
+class ProgrammingLanguage:
+    """Represent information about a programming language."""
+
+    def __init__(self, name: str, typing: str, reflection: bool, year: int, pointer_arithmetic: bool):
+        """Construct a ProgrammingLanguage from the given values.
+
+        name: string
+        typing: string
+        reflection: boolean
+        year: int
+        pointer_arithmetic: boolean
+        """
+        self.name = name
+        self.typing = typing
+        self.reflection = reflection
+        self.year = year
+        self.pointer_arithmetic = pointer_arithmetic
+
+    def __repr__(self):
```

Comment:
You can also choose to add a __str__ method to make it more readable for plain output.

Reply:
Yeah I did it that way since the `__repr__` method is used if a `__str__` method is not present